std/experimental/logger/filelogger.d

```diff
@@ -21,17 +21,36 @@ class FileLogger : Logger
       file can not be opened for writting an exception will be thrown.
       lv = The $(D LogLevel) for the $(D FileLogger). By default the
       $(D LogLevel) for $(D FileLogger) is $(D LogLevel.all).
+      createFileNameFolder = if true and fn contains a folder name, this
```

Comment:
Nits: 
- How about using a Flag instead of a boolean?
- How about naming it to sth. similar to `make -p`
`makeParentDirectories = If set to `true`, parent directories will be created as needed